snippets/facets.liquid

```diff
@@ -381,6 +381,7 @@
   {%- endif -%}
   {% comment %}  Drawer and mobile filter {% endcomment %}
   <menu-drawer class="mobile-facets__wrapper{% if filter_type == 'horizontal' or filter_type == 'vertical' %} medium-hide large-up-hide{% endif %}" data-breakpoint="mobile">
+    <h2 class="visually-hidden" id="verticalTitle">{{ 'products.facets.filter_by_label' | t }}</h2>
```

Comment:
Hmm isn't this going to create a duplicate ID since it will have one on mobile for the menu drawer and one for the desktop version 🤔